lib/orderbook_sync.js

```diff
@@ -30,20 +30,39 @@ class OrderbookSync extends WebsocketClient {
       );
     }
 
-    this.productIDs.forEach(productID => {
-      this._queues[productID] = [];
-      this._sequences[productID] = -2;
-      this.books[productID] = new Orderbook();
-    });
+    this.productIDs.forEach(this.newProduct, this);
+  }
+
+  newProduct(productID) {
+    this._queues[productID] = [];
+    this._sequences[productID] = [];
+    this.books[productID] = new Orderbook();
   }
```

Comment:
This method probably deserves to be prefixed with a `_` to designate that it represents more internal/private behavior (i.e. `_newProduct()`).